tests/integration_test.go

```diff
@@ -96,6 +99,9 @@ func generateBlocksAsync() {
 }
 
 func TestXMRTaker_Discover(t *testing.T) {
+	if os.Getenv(generateBlocksEnv) != falseStr {
+		generateBlocks(64)
+	}
```

Comment:
why do we need to generate blocks here?

Reply:
I was testing and I got an insufficient balance failure, so I went in and added some block generation to every test that was required a non-zero balance.  Thinking about your comment though, I'll take another pass at those changes in my next PR. Maybe we should eliminate the balance pre-generation before any tests are run?